templatestore/frontend/src/components/Dropdowncomp.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from '../style/templateScreen.less';
+export default function Dropdowncomp(props) {
+    return (
+        <div className={styles.teAttributesRow}>
+            <div className={styles.teAttributesCell}>
+                <select
+                    name="urlKey"
+                    value={props.urlKey}
```

Comment:
Can we have a better name here? Maybe `field`?

Reply:
Can you please elaborate which name are you referring to?

Reply:
I've used urlKey because we're extracting it from urlKeyList.

Reply:
Okay, sounds good!